pkg/providers/nutanix/template.go

```diff
@@ -415,6 +415,10 @@ func buildTemplateMapMD(clusterSpec *cluster.Spec, workerNodeGroupMachineSpec v1
 		values["additionalCategories"] = workerNodeGroupMachineSpec.AdditionalCategories
 	}
 
+	if len(workerNodeGroupMachineSpec.GPUs) > 0 {
+		values["GPUs"] = workerNodeGroupMachineSpec.GPUs
+	}
+
```

Comment:
Are GPUs only supported for worker nodes?
If so, we should add a validation to block GPU configuration on `NutanixMachineConfig` corresponding to control plane and etcd nodes.

Reply:
Fixed